### ISSUE TITLE
Adding Localization Philosophy

### ISSUE THREAD

**Issue by anhourandaquarter** [wordplaydev/wordplay#587]
### What are you trying to do that you can't?

There are a lot of language choices that go into a localization that might not necessarily be reflected in the final localization. Right now, I'm just logging these choices in to a Discord thread, but I'd like to also share these design choices with Wordplay users.

### What is your idea?

 I think it'd be great to add a place on the Wordplay landing page to include the design/translation philosophies behind each localization. Each of the localizations can add their own pages for their own design philosophies there (whether completed or still in development). 

I think this could also be a great way for folks to keep up with the development stages of the forthcoming localizations.

**Comment by amyjko** [comment]
Excellent idea. We just need a design for this and then we can start capturing and sharing this rationale.

**Comment by anhourandaquarter** [comment]
I'd like to contribute to this further from a translator's perspective, and I'd greatly appreciate having collaborators on the design aspect!

**Comment by amyjko** [comment]
Great! Assigned.

**Comment by shreyam47** [comment]
@amyjko cleanup - inactivity

**Comment by amyjko** [comment]
@infinite0x20 Are you still working on this?

**Comment by infinite0x20** [comment]
Yes, I am still working on drafting an initial localization philosophy for Korean. Would love to find collaborators to design and implement the feature, though!

**Comment by amyjko** [comment]
That's great news, thanks @infinite0x20! I'm always looking for volunteers, so I'll be sure to share if any want to contribute.

**Comment by amyjko** [comment]
Hi! I'm Amy's GitHub bot. @infinite0x20 — Are you still working on this issue? Can you post a status update, and if you won't finish it, can you unassign yourself? We'll unassign you in a week if we don't get a reply.

**Comment by infinite0x20** [comment]
I'd like to still work on this issue but would prefer to have other collaborators! For now I have been focusing most of my time on the Korean localization.

**Comment by amyjko** [comment]
Sounds good @infinite0x20! It would definitely be ideal to have other people contribute to this. Perhaps reach out to others who've been localizing and see if they want to work together?

**Comment by amyjko** [comment]
Hi! I'm Amy's GitHub bot. @infinite0x20 — Are you still working on this issue? Can you post a status update, and if you won't finish it, can you unassign yourself? We'll unassign you in a week if we don't get a reply.

**Comment by infinite0x20** [comment]
Taking a hiatus from this for the summer but will return to it in autumn!

**Comment by amyjko** [comment]
Status check: most of this is now built, via the per-locale `guidance` field added in #1154.

**What exists**

- Every locale has a top-level `guidance` field (`LocaleText.ts`), written in that locale's *own* language. It is the one field that isn't a translation: the locale verifier skips it, so it is never machine translated and never counted unwritten.
- It's authored and read in two places: the **About** tab of the `/localize` workspace, and the localization-mode panel that sits above every page. Edits save like any other string and land in the contributor's submission bundle, so a philosophy is versioned with the locale rather than living in a Discord thread.
- The landing page links to `/localize`.
- It's also *used*: each locale's guidance is fed to the machine translator as its writing conventions, so a locale's stated philosophy shapes future translations.
- Five locales have written one so far: es-MX, ko-KR, pt-PT, zh-CN, zh-TW.

**What's still missing from this issue's ask**

1. It isn't public. `/localize` requires sign-in, as does the localization-mode panel, so a Wordplay *user* can't read a locale's philosophy — only a signed-in contributor can, and only for whichever locale is currently active. There's no page that lists the localizations and their philosophies side by side.
2. There's no view of a localization's development stage beyond the "draft" badge in the language chooser.

Leaving this open for those two pieces — a public, browsable version of what `guidance` already captures.

**Comment by amyjko** [comment]
Closing: `/localize` is the home for this. Each locale's `guidance` lives there, written in that locale's own language, versioned with the locale, and editable in place — which covers sharing the design and translation philosophy behind each localization. Thanks @infinite0x20 for proposing it.
